web/packages/shared/components/UnifiedResources/unifiedStyles.css

```diff
@@ -20,16 +20,9 @@
     TODO (avatus): DELETE if we ever upgrade to v6
 */
 .ContainerContext {
-  min-width: 800px;
```

Comment:
Removed because of https://github.com/gravitational/teleport/pull/33916#discussion_r1374582777

@avatus was there a reason for having it? From what I see, a parent element in Web UI specifies min width that is larger than 800px.
And even if it didn't, the min width of the shared component would be 400px (because of `minmax(400px, 1fr)` which still should not break the layout.

Reply:
Probably a left over. Your change works fine for me locally